frontend/default/routes.py

```diff
@@ -11,9 +14,29 @@ def index():
 
 @default_bp.errorhandler(404)
 def not_found(error):
-    return render_template("404.html"), 404
+    current_app.logger.error(f"Encountered 404: {error}")
+    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
+    return render_template(
+        "404.html",
+        contact_us_phone=round_data.contact_details["phone"],
+        contact_us_email_address=round_data.contact_details["email_address"],
+        contact_us_text_phone=round_data.contact_details["text_phone"],
+        opening_time=round_data.support_availability["time"],
+        opening_days=round_data.support_availability["days"],
+        closed=round_data.support_availability["closed"]
```

Comment:
Could this just be reduced to `round=round` and get the Round object above?

Reply:
Or maybe 
`contact_details=round.contact_details`
`support_availability=round.support_availability` better?

Reply:
Need to handle potential error from `get_round_data`